### DISCUSSION TITLE
Version 1.2.0

### DISCUSSION THREAD

**Discussion by ahadas** [mucommander/mucommander#872] [General]
I think we have enough content for a release
I'd like to try to address [the two remaining issues which are currently targeted to 1.2.0](https://github.com/mucommander/mucommander/milestone/13) (I have a fix for #773 that needs to be polished a bit and I can create a version with more information to check #832 ).
@pskowronek @hajdam, anything you work on that you'd also like to get in to 1.2.0 (#823 ?)?

**Comment by pskowronek** [comment]
@ahadas nope, im fine. Docker notifications PR will do in the next release.

**Comment by hajdam** [comment]
Nothing necessary from me I hope.
I'm aware of some issues with image viewer (some rendering issue when zoomed image resolution is greater than 60k pixels).
Also planned to add undo/redo support for binary editor, but that can wait for sure...

**Comment by ghost** [comment]
> I can create a version with more information to check #832

Looking forward to that special debug version. It would be great if #832 could be fixed before the release of v1.2.0. The need to adjust the settings again and again makes the whole new editor more or less useless to me.

**Reply by ahadas** [reply]
@Andreas0602 FYI I've just created a nightly build with fixes for #812 and #874

**Comment by ghost** [comment]
I can confirm that both fixes are working well. Thank you very much again!

**Comment by ahadas** [comment]
Version 1.2.0 [has been released](https://github.com/mucommander/mucommander/releases/tag/1.2.0-1), thanks to everyone who has contributed to this release!
